Sources/SpeziOnboarding/ConsentView.swift

```diff
@@ -197,6 +228,132 @@ public struct ConsentView<ContentView: View, Action: View>: View {
     }
 }
 
+
+/// Extension of ``ConsentView`` enabling the export of the signed consent page in the onboarding flow.
+extension ConsentView {
+    /// Creates a view representation of the consent content, ready for PDF export via SwiftUIs `ImageRenderer`.
+    /// At the moment, this is
+    ///
+    /// This function constructs a view for presenting the markdown consent form. It combines the
+    /// given markdown and the user's signature with details such as the date of export. It can be
+    /// used to create exportable PDF documents of the consent form.
+    ///
+    /// - Parameters:
+    ///   - markdown: The markdown consent content as an `AttributedString`.
+    ///
+    /// - Returns: A SwiftUI `View` representation of the consent content and signature.
+    ///
+    /// - Note: This function avoids the use of asynchronous operations.
+    /// Asynchronous tasks are incompatible with SwiftUI's `ImageRenderer`,
+    /// which expects all rendering processes to be synchronous.
+    func exportBody(markdown: AttributedString) -> some View {
+        VStack {
+            HStack {
+                Spacer()
+                
+                Text("Exported: \(DateFormatter.localizedString(from: Date(), dateStyle: .medium, timeStyle: .short))")
+                    .font(.caption)
+            }
+            .padding()
+            
+            Text("Spezi Consent")
+                .font(.title)
```

Comment:
Good idea with the title. Thinking about reusability, I think this should either be configurable or removed. It might make sense

Reply:
Yes, definitely also wanted to do that, but I didn't want to blow up the signature of the View too much. However, after you gave me the "Go" to heavily simplify the API of that view, this is definitely a good idea 👍